packages/api/src/command/medical/document/document-conversion-status.ts

```diff
@@ -45,112 +54,88 @@ export async function calculateDocumentConversionStatus({
+        totalDocsConverted,
+        successfulConversions,
+        failedConversions,
       },
-      requestId,
-      source,
     });
+  }
 
-    const externalData =
-      source === MedicalDataSource.COMMONWELL
-        ? getCWData(updatedPatient.data.externalData)
-        : getCQData(updatedPatient.data.externalData);
-
-    const globalTriggerConsolidated =
-      updatedPatient.data.documentQueryProgress?.triggerConsolidated;
-    const hieTriggerConsolidated = externalData?.documentQueryProgress?.triggerConsolidated;
-
-    const isGlobalConversionCompleted = isProgressStatusValid({
-      documentQueryProgress: updatedPatient.data.documentQueryProgress,
-      progressType: "convert",
-      status: "completed",
-    });
-    const isHieConversionCompleted = isProgressStatusValid({
-      documentQueryProgress: externalData?.documentQueryProgress,
-      progressType: "convert",
-      status: "completed",
+  if (
+    (hieTriggerConsolidated && isHieConversionCompleted) ||
+    (globalTriggerConsolidated && isGlobalConversionCompleted)
+  ) {
+    log(
+      `Kicking off getConsolidated for patient ${updatedPatient.id} - hie: ${hieTriggerConsolidated} global: ${globalTriggerConsolidated}`
+    );
+    // intentionally async
+    recreateConsolidated({
+      patient: updatedPatient,
+      conversionType: "pdf",
+      context: `Post-DQ getConsolidated ${source}`,
+      requestId,
+      isDq: true,
     });
-
-    if (isHieConversionCompleted) {
-      const startedAt = updatedPatient.data.documentQueryProgress?.startedAt;
-      const convert = updatedPatient.data.documentQueryProgress?.convert;
-      const totalDocsConverted = convert?.total;
-      const successfulConversions = convert?.successful;
-      const failedConversions = convert?.errors;
-
-      analytics({
-        distinctId: cxId,
-        event: EventTypes.documentConversion,
-        properties: {
-          requestId,
-          patientId,
-          hie: source,
-          duration: elapsedTimeFromNow(startedAt),
-          totalDocsConverted,
-          successfulConversions,
-          failedConversions,
-        },
-      });
-    }
-
-    if (
-      (hieTriggerConsolidated && isHieConversionCompleted) ||
-      (globalTriggerConsolidated && isGlobalConversionCompleted)
-    ) {
-      log(
-        `Kicking off getConsolidated for patient ${updatedPatient.id} - hie: ${hieTriggerConsolidated} global: ${globalTriggerConsolidated}`
-      );
-      // intentionally async
-      recreateConsolidated({
-        patient: updatedPatient,
-        conversionType: "pdf",
-        context: `Post-DQ getConsolidated ${source}`,
-        requestId,
-        isDq: true,
-      });
-    } else if (isGlobalConversionCompleted) {
-      // intentionally async
-      recreateConsolidated({
-        patient: updatedPatient,
-        context: "Post-DQ getConsolidated GLOBAL",
-        requestId,
-        isDq: true,
-      });
-    }
-  } else {
```

Comment:
Removing this whole section after I validated we don't run this any longer - [logs for 2 weeks](https://us-west-1.console.aws.amazon.com/cloudwatch/home?region=us-west-1#logsV2:log-groups/log-group/APIInfrastructureStack-APIFargateServiceTaskDefAPIServerLogGroup35E9518F-7A2oRfufKv3Z/log-events$3FfilterPattern$3D$2522Running+calculateDocumentConversionStatus+without$2522$26start$3D-1209600000).